test/ngAnimate/animateSpec.js

```diff
@@ -168,7 +168,7 @@ describe('animations', function() {
       inject(function($animate, $rootScope) {
         $animate.enter(element, parent);
         $rootScope.$digest();
-        expect(capturedAnimation).toBeFalsy();
+        expect(capturedAnimation).toBe(null);
```

Comment:
I'd put those changes in a separate commit as they don't really have a lot in common with the original one.